### PR TITLE
#44 Updating user avatar

### PR DESCRIPTION
**Pull request by Werfit**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -40,12 +40,16 @@
...
+    "@types/aws-sdk": "^2.7.0",
     "@types/bcrypt": "5.0.0",
+    "@types/express-fileupload": "^1.2.2",
     "@types/jsonwebtoken": "8.5.8",
+    "aws-sdk": "^2.1073.0",
     "bcrypt": "5.0.1",
     "cors": "2.8.5",
     "dotenv": "14.3.2",
     "express": "4.17.2",
+    "express-fileupload": "^1.3.1",
```
Comment (aleksandr-kuharenko) [inline backend/package.json]:
```suggestion
    "@types/aws-sdk": "2.7.0",
    "@types/bcrypt": "5.0.0",
    "@types/express-fileupload": "1.2.2",
    "@types/jsonwebtoken": "8.5.8",
    "aws-sdk": "2.1073.0",
    "bcrypt": "5.0.1",
    "cors": "2.8.5",
    "dotenv": "14.3.2",
    "express": "4.17.2",
    "express-fileupload": "1.3.1",
```

File: backend/src/api/routes/user-route.ts [hunk 2]
```diff
@@ -1,18 +1,23 @@
 import { Request, Router } from 'express';
+// import { UploadedFile,,,/ } from 'express-fileupload';
```
Comment (aleksandr-kuharenko) [inline backend/src/api/routes/user-route.ts]:
```suggestion
```

File: backend/src/common/utils/upload-image.util.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import AWS from 'aws-sdk';
+// import { Express } from 'express';
+// import { UploadedFile } from 'express-fileupload';
```
Comment (aleksandr-kuharenko) [inline backend/src/common/utils/upload-image.util.ts]:
```suggestion
```

File: backend/src/server.ts [hunk 4]
```diff
@@ -2,6 +2,7 @@ import cors from 'cors';
 import path from 'path';
 import express, { Express } from 'express';
 import { createConnection } from 'typeorm';
+// import fileUpload from 'express-fileupload';
```
Comment (aleksandr-kuharenko) [inline backend/src/server.ts]: remove all comments

File: package.json [hunk 5]
```diff
@@ -55,7 +55,10 @@
     "shared/**/*.{ts,tsx}": "npm run lint:shared"
   },
   "dependencies": {
+    "@types/multer": "^1.4.7",
     "axios": "^0.25.0",
+    "multer": "^1.4.4",
+    "react-avatar": "^4.0.0",
```
Comment (aleksandr-kuharenko) [inline package.json]: lock libs

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Comment by nikita-remeslov** [comment]
Resolve conflicts

**Pull request merged**